package.json

```diff
@@ -9,9 +9,9 @@
   "author": "akiroz",
   "license": "Apache-2.0",
   "dependencies": {
-    "skygear": "^0.23.0"
+    "skygear": "^1.0.0"
```

Comment:
Since this PR requires a newer version of skygear js sdk to work for the newer implementation of `skygear.pubsub.reconfigure()`, this should be specified to that version.